ansible/playbooks/roles/opensearch/defaults/main.yml

```diff
@@ -61,3 +61,4 @@ ports:
   http:      9200  # defaults to range but we want static port
   transport: 9300  # defaults to range but we want static port
 log4j_file_name: apache-log4j-2.17.1-bin.tar.gz
+systemctl_path: /etc/systemd/system
```

Comment:
IMO this can be hard-coded. The var's name is misleading for me - `systemctl` is CLI for systemd.

Reply:
It's reworked now as U suggested.